pyart/graph/radardisplay.py

```diff
@@ -1019,6 +1050,44 @@ def _label_axes_vpt(self, axis_labels, ax):
         else:
             ax.set_ylabel(y_label)
 
+    def _set_vpt_time_axis(self, ax, dForm=None, tz=None,
+                           date_MinTicker=None):
+        """ Set the x axis as a time formatted axis.
+
+        Parameters
+        ----------
+        ax : Matplotlib axis instance
+            Axis to plot. None will use the current axis.
+        dForm : str
```

Comment:
Okay, renamed ```dForm```, kept ```tz``` the same (unless you want that more explicit). ```date_MinTicker``` was removed via comment below.